deeplearning4j-keras/src/test/java/org/deeplearning4j/keras/DeepLearning4jEntryPointTest.java

```diff
@@ -38,12 +52,46 @@ public void shouldFitTheSampleSequentialModel() throws Exception {
         // fall through - the rule will fail the test execution if an exception is thrown
     }
 
+    private Path prepareLabels(String dataSet) throws IOException {
+        return prepareDataSet(dataSet, "labels");
+    }
+
+    private Path prepareFeatures(String dataSet) throws IOException {
+        return prepareDataSet(dataSet, "features");
+    }
+
+    private Path prepareDataSet(String dataSet, String batchesSubset) throws IOException {
+        Set<String> batchFiles = listBatchFiles(dataSet + "/" + batchesSubset);
+        final Path tempDirectory = Files.createTempDirectory("dl4j-" + batchesSubset);
+
+        for (String batchFile : batchFiles) {
+            String batchFileName = Paths.get(batchFile).getFileName().toString();
+            copyClasspathResourceToFile(batchFile, targetFile(tempDirectory, batchFileName));
+        }
+        return tempDirectory;
+    }
+
+    private File targetFile(Path tempFeaturesDirectory, String batchFileName) {
+        return Paths.get(
+                tempFeaturesDirectory.toFile().getAbsolutePath().toString(),
+                batchFileName
+        ).toFile();
+    }
+
+    private Set<String> listBatchFiles(String classpathDirectory) {
+        return new Reflections(classpathDirectory, new ResourcesScanner()).getResources(endsWith(".h5"));
+    }
```

Comment:
Not sure why you'd use reflections here. This shouldn't have to be on the classpath?